kbfsgit/runner.go

```diff
@@ -1435,6 +1435,16 @@ func (r *runner) pushAll(ctx context.Context, fs *libfs.FS) (err error) {
 		fmt.Sprintf("Preparing and %s packed refs", verb), "pushprefs")
 }
 
+func dstNameFromRefString(refStr string) plumbing.ReferenceName {
+	start := strings.Index(refStr, ":") + 1
+	dst := refStr[start:]
+	return plumbing.ReferenceName(dst)
+}
+
+func dstNameFromRefSpec(refspec gogitcfg.RefSpec) plumbing.ReferenceName {
+	return dstNameFromRefString(refspec.String())
```

Comment:
`return refspec.Dst("")`, after which you don't need the above.

Reply:
Hmm good point.  I wonder why I had it fully implemented before.  Maybe I was worried about the wildcard behavior?  But I don't think the remote helper gets wildcard specs over the local protocol anymore, so it's probably ok to do it the way you suggested.